pages/price-feeds/best-practices.mdx

```diff
@@ -70,6 +73,18 @@ This analogy suggests two simple solutions to races:
    The protocol can require a short delay between the two transactions, and the user's request gets fulfilled at the Pyth price as of the second transaction.
    This technique gives the protocol extra time to observe price changes, giving it a head start in the latency race.
 
+### Latency Mitigations for Derivative Protocols
+
+To mitigate the risk of latency, derivative protocols should consider the following strategies:
+
+1. **Use a Delayed Settlement**: Derivative protocols can introduce a delay between the time a contract is executed and the time it is settled. This delay gives the protocol time to observe price changes and reject transactions that are based on manipulated prices.
```

Comment:
probably you can say:
... between the time an order is created and the time it is executed

Reply:
Also, "manipulated price" is wrong. You could say "... reject trades that profit over latency".